### ISSUE TITLE
Missing tertiary and highway links in cities

### ISSUE THREAD

**Issue by atorger** [atorger/nvdb2osm#19]
There's a bug causing gatutyp layer always override KLASS, meaning that we get residential for all but the largest roads inside cities. Link is not processed at all currently.

**Comment by atorger** [comment]
I've now tested the highway _link trick. Neat.

However, it is a bit "dangerous". What it does is subtracting FPV-KLASS from DoU-KLASS, and looking at what's left. It can only be used on roads that are included in FPV-klass, ie primary and up, and it trusts that DoU-KLASS includes links and that FPV-KLASS does not, so the difference = links.

I tested it on Malmö and it sort of works, but not quite. I'm hesitant to use it due to that it's just a side-effect how these layers happen to be delivered, rather than a layer actually pointing out links. It also is of limited value to include link tagging as ~100% of all these roads are already mapped, and 50+% will not match on highway type anyway so doing import there will mostly only be about refining geometry. A time saver for import is probably just to drop the highway tag (keep the others) and replace geometry to merge the rest.

So for now I won't include highway links in my translation..
